tests/test_schemas.py

```diff
@@ -83,19 +84,19 @@ def getInvalidValue(self, cls, fieldName):
         Returns a value that should trigger a schema validation failure.
         """
         field = self.getAvroSchema(cls, fieldName)
-        t = field.type
-        if isinstance(t, avro.schema.UnionSchema):
-            t = t.schemas[1]
-        if isinstance(t, avro.schema.MapSchema):
+        typ = field.type
```

Comment:
Yeah, I'm not really bothered about it. It's up to you if you want to bring it in.